README.md

```diff
@@ -171,13 +171,13 @@ katello-service restart
 ```
```

Comment:
Note: the line above this still uses `katello-service` but that's been replaced with `satellite-maintain restart`. Also don't think you should recommend 1024 bits DH params these days.

Reply:
Have replaced katello-service with satellite-maintain.
For the 1024 bits have left it as is for now, feel that change should be included as a larger update to the module docs.

Reply:
@david22swan should line 169 also have been modified to specify satellite-maintain instead?

Reply:
@michaeltlombardi 
Yeh, that change was intentional